### PR TITLE
new feature and minor code refactoring

### PR DESCRIPTION
**Pull request by robophil**
1. Set color for tag from tag-me attribute  #2. refactor data_init to use tenary operator

### CODE REVIEW

**Comment by robophil** [comment]
just add attribute tagColor="red" from directive to set background color of tags from directive

File: templates/input.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 </div>
 
 <ul class="tag-typehead" ng-show="input.length && typehead && results.length " >
-    <li class="animate-repeat" ng-repeat="item in data | filter:input as results">
+    <li class="animate-repeat" ng-repeat="item in data | filter:input as results" ng-style="tagStyleColor">
```
Comment (theoomoregbee) [inline templates/input.html:15]: that part is the type head styling , not really the tag styling ,

**Comment by theoomoregbee** [comment]
if setting the tag color dynamically instead of using css to overide it like inside the custom css instead , so when it is active or selected the styling can be done anyhow u want via ur own css rule , or what do u think?

**Comment by robophil** [comment]
Yeah overriding the css rule with yours is most appropriate.
For a quick fix without having to dive into css, that would be nice, and situations where you have multiple <tag-me> directives, it could come in handy

File: templates/input.html (hunk 1 above)
Comment (robophil) [inline templates/input.html:15]: head styling?

Comment (theoomoregbee) [inline templates/input.html:15]: yh, type head , when you type the sugestion that is coming out is the type head

Comment (robophil) [inline templates/input.html:15]: Would test it out.

**Pull request closed without merging**